lib/api_local.ml

```diff
@@ -88,6 +89,12 @@ module Slack : Api.Slack = struct
   let send_auth_test ~ctx:_ () =
     Lwt.return
     @@ Ok ({ url = ""; team = ""; user = ""; team_id = ""; user_id = "test_slack_user" } : Slack_t.auth_test_res)
+
+  let get_thread_permalink ~ctx:_ (thread : State_t.slack_thread) =
+    Lwt.return_some
+    @@ Printf.sprintf "https://monorobot.slack.com/archives/%s/p%s?thread_ts=%s&cid=%s" thread.cid
```

Comment:
is it the correct url?

Reply:
no, this is a dummy url. The structure is correct, but the domain is not. i didn't want to use the real one. This is only used for tests, I don't think it matters.

Reply:
Can put as a comment in code please?